File: Assignment5/program1.c
///////////////////////////////////////////////////////////////////////////////////
//
//    File namev:  program1.c
//    Description: Accept the number from the user and print the number of $ and *
//    Author :     Mayuresh Kaluskar
//    Date :       14/05/2025
//
////////////////////////////////////////////////////////////////////////////////////




#include <stdio.h>
void Display(int iNo)
{
  int iCnt =0;

  if(iNo<0)
  {
    iNo = -iNo;
  }

  for(int iCnt=1;iCnt<=iNo;iCnt++)
  {
    printf("$  * ");
  }
  

}

int main()
{
    int iVal = 0;
    
    printf("ENTER THE number :");
    scanf("%d",&iVal);

  Display(iVal);
  
  return 0;
}

File: Assignment11/program1.java
import java.util.Scanner;

class DigitDisplay
{
    public void Display(int iNo)
    {
      char ch = 'A';
      for(int i = 1;i<=iNo;i++,ch++)
      {
          System.out.print(ch+"\t");
      }
    }
}

class program1
{
    public static void main(String[] args) {
        int iVal = 0;

        Scanner sobj =new Scanner(System.in);

        System.out.println("Enter the number");
        iVal = sobj.nextInt();
         
        DigitDisplay dobj = new DigitDisplay();

        dobj.Display(iVal);
        
    }
}